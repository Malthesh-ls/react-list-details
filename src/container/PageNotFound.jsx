const PageNotFound = () => {
  return (
    <div className="section">
      <div className="alignCenter">Page Not Found</div>
    </div>
  )
}

export default PageNotFound