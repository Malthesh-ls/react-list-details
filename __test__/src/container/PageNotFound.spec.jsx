import renderer from 'react-test-renderer';
import PageNotFound from '../../../src/container/PageNotFound';

describe('PageNotFound Tests', () => {
  it('should match snapshot', () => {
    const snapshot = renderer
      .create(<PageNotFound />)
      .toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});