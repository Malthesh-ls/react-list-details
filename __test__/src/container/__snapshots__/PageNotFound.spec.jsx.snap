// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Tests should match snapshot 1`] = `
<div
  className="section"
>
  <div
    className="alignCenter"
  >
    Page Not Found
  </div>
</div>
`;
